api/write/cron.php

```diff
@@ -0,0 +1,14 @@
+<?php
+$paths = yaml_parse(file_get_contents(__DIR__ . '/../../config/paths.yml'));
```

Comment:
Again, can use api_read.php and expose a path method instead of duplicating here...